zerver/lib/markdown/help_settings_links.py

```diff
@@ -122,6 +122,9 @@ def getMarkdown(setting_type_name: str, setting_name: str, setting_link: str) ->
         setting_type_name=setting_type_name,
         setting_reference=setting_name,
     )
+    if setting_name == "Bots":
```

Comment:
Let's add a comment in this block explaining that we need special text here because Bots is a label that appears in both tabs.